boards/shields/lcd_par_s035/lcd_par_s035_8080.overlay

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2024 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/dt-bindings/mipi_dbi/mipi_dbi.h>
+
+/{
+	chosen {
+		zephyr,display = &st7796s;
+	};
+};
+
+&zephyr_flexio_lcd {
```

Comment:
We should name the nodelabel `nxp_flexspi_lcd`- Zephyr implies this node label is generic, which it is not